lib/emoji_dictionary.dart

```diff
@@ -31,3 +31,12 @@ class EmojiDictionary {
     'snacks': '🍿'
   };
 }
+
+// TODO this might replace the dictionary, though it looks like it is limited to english
```

Comment:
this is the important part

Reply:
ah fuck the comments broke the build, guess rm before merge, still up for review tho